sources/Cargo.toml

```diff
@@ -55,6 +55,7 @@ members = [
     "settings-migrations/v1.23.0/nvidia-container-runtime-settings",
     "settings-migrations/v1.23.0/kubelet-device-plugins-metadata",
     "settings-migrations/v1.23.0/kubelet-device-plugins-settings",
+    "settings-migrations/v1.23.0/kubernetes-service-config",
```

Comment:
Since, we released v1.23.0 yesterday, you will have to aim for 1.24.0 or 1.24.1. You will have to rebase and add the migration files to the appropriate directory.